test/integration/cli/security-cve-create.t

```diff
@@ -19,5 +19,5 @@ Create test input
 Run test case
 
   $ tbx security cve create cves.jsonl 2> /dev/null | grep .
```

Comment:
Formatting, would make the test more readable, e.g.

`tbx security cve create cves.jsonl 2> /dev/null | grep . | python -m json.tool`